components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.ts

```diff
@@ -182,4 +189,21 @@ export class ReportingNodeComponent implements OnInit, OnDestroy {
   formatDaysAgo(timestamp) {
     return moment(timestamp).fromNow();
   }
+
+  private getReports(reportQuery: ReportQuery): Observable<ReportCollection>  {
+    return this.statsService.getReportsWithPages(reportQuery,
+      {sort: 'latest_report.end_time', order: 'DESC'}, this.page, this.pageSize)
+    .pipe(first());
```

Comment:
It is equivalent, but I think it is more clear if the `.pipe(first())` is on the recipient, so it is directly attached to the `subscribe`.